server/src/main/java/org/elasticsearch/indices/breaker/HierarchyCircuitBreakerService.java

```diff
@@ -622,6 +622,12 @@ public MemoryUsage overLimit(MemoryUsage memoryUsed) {
                         long now = timeSupplier.getAsLong();
                         this.lastCheckTime = now;
                         allocationDuration = now - begin;
+                    } else {
+                        logger.info(
```

Comment:
I wonder if we need to throttle this in order to not flood the log with those? Perhaps include an attempt number (reset whenever we actually trigger the GC) and then only output it if `attempNo < 10 || Long.bitCount(attemptNo) == 1` or so?

Reply:
I merged this line with the ones below and added throttling to avoid flooding the log.